romancal/assign_wcs/assign_wcs_step.py

```diff
@@ -0,0 +1,73 @@
+"""
+Assign a gWCS object to a science image.
+
+"""
+import logging
+
+from astropy import coordinates as coord
+from astropy import units as u
+import gwcs.coordinate_frames as cf
+
+from gwcs.wcs import WCS, Step
+
+from roman_datamodels import datamodels as rdm
+from romancal.stpipe import RomanStep
+from . import pointing
+from .utils import wcs_bbox_from_shape
+
+
+log = logging.getLogger(__name__)
+log.setLevel(logging.DEBUG)
+
+
+__all__ = ["AssignWcsStep", "load_wcs"]
+
+
+class AssignWcsStep(RomanStep):
+    """ Assign a gWCS object to a science image.
+    """
+
+    reference_file_types = []
+
+    def process(self, input_model):
+        reference_file_names = {}
+        with rdm.open(input_model) as input_model:
```

Comment:
Style question: do we want to open the model name as the same name?

Reply:
there's no rule against this

Reply:
Hai, is a question of style. Resolving this for merge.